website/docs/r/elastic_cloud_elasticsearch.html.markdown

```diff
@@ -41,6 +41,10 @@ The following arguments are supported:
 
 * `sku_name` - (Required) Specifies the name of the SKU for this Elasticsearch. Changing this forces a new Elasticsearch to be created.
 
+**NOTE:** The SKU depends on the Elasticsearch Plans available for your account and is a combination of PlanID_Term.\
```

Comment:
Wondering why do we need the tailing back slash here?

Reply:
Answered at https://github.com/hashicorp/terraform-provider-azurerm/pull/23863#discussion_r1389184081